docs/source/tutorial/package_testing.rst

```diff
@@ -39,23 +41,46 @@ Near the end of the log, you should see something like this:
     2019-05-01 13:08:40,563 - DataIngestPipeline - INFO - ❌ Count Analysis Failed!
     See /path/to/my_study/logs/counts_for_ingest.log for details
 
-Inside of that ``counts_for_ingest.log`` file, you should some things that look
+Inside of that ``counts_for_ingest.log`` file, you should something that look
 like::
 
-    EXPECTED COUNT CHECKS
-    +----------------+------------+---------+---------+
-    | key            |   expected |   found | equal   |
-    |----------------+------------+---------+---------|
-    | FAMILY|ID      |          2 |       2 | ✅      |
-    | PARTICIPANT|ID |          5 |       9 | ❌      |
-    | BIOSPECIMEN|ID |         10 |      16 | ❌      |
-    +----------------+------------+---------+---------+
+    ❌ Column names not equal between ExtractStage and TransformStage
+
+    +--------------------------+---------------------+
+    | In ExtractStage          | In TransformStage   |
+    |--------------------------+---------------------|
+    | PARTICIPANT|MOTHER_ID    |                     |
+    | PARTICIPANT|FATHER_ID    |                     |
+    | PHENOTYPE|EVENT_AGE_DAYS |                     |
+    | PHENOTYPE|NAME           |                     |
+    | PHENOTYPE|OBSERVED       |                     |
+    +--------------------------+---------------------+
+
+It looks like the tests are failing because the columns in the extract stage
+output DataFrames are different than the TransformStage's output DataFrames.
```

Comment:
inconsistency between "extract stage" non-possessive, lowercase, two words and "TransformStage's" possessive, camel case, one word